packages/web/src/components/track/desktop/TrackTile.test.tsx

```diff
@@ -0,0 +1,164 @@
+import { developmentConfig } from '@audius/sdk'
+import { http, HttpResponse } from 'msw'
+import { Route, Routes } from 'react-router-dom-v5-compat'
+import { describe, expect, it, beforeAll, afterEach, afterAll } from 'vitest'
+
+import { mswServer, render, screen } from 'test/test-utils'
+
+import { TrackTileSize } from '../types'
+
+import { TrackTile } from './TrackTile'
+
+const { apiEndpoint } = developmentConfig.network
+
+const testUser = {
+  id: 'ML51L',
+  handle: 'test-user',
+  name: 'Test User',
+  is_verified: false,
+  is_deactivated: false,
+  artist_pick_track_id: null
+}
+
+const testTrack = {
+  id: '7eP5n',
+  track_id: '7eP5n',
+  user_id: 'ML51L',
+  genre: 'Electronic',
+  title: 'Test Track',
+  user: testUser,
+  duration: 180,
+  repost_count: 5,
+  favorite_count: 10,
+  comment_count: 15,
+  permalink: '/test-user/test-track',
+  is_delete: false,
+  is_stream_gated: false,
+  is_unlisted: false,
+  has_current_user_reposted: false,
+  has_current_user_saved: false,
+  preview_cid: 'QmTestPreviewCid',
+  _co_sign: null,
+  is_owned_by_user: false,
+  is_scheduled_release: false,
+  is_available: true,
+  is_downloadable: true,
+  play_count: 1,
+  artwork: null,
+  followee_reposts: [],
+  followee_favorites: [],
+  track_segments: [],
+  field_visibility: {}
+  // Add any other required fields with default values
+}
+
+function renderTrackTile(overrides = {}) {
+  mswServer.use(
+    http.get(`${apiEndpoint}/v1/full/tracks`, ({ request }) => {
+      const url = new URL(request.url)
+      const id = url.searchParams.get('id')
+      if (id === '7eP5n') {
+        return HttpResponse.json({ data: [{ ...testTrack, ...overrides }] })
+      }
+      return new HttpResponse(null, { status: 404 })
+    }),
+    http.get(`${apiEndpoint}/v1/events/entity`, () => {
+      return HttpResponse.json({ data: [] })
+    })
+  )
+
+  return render(
+    <Routes>
+      <Route
+        path='/'
+        element={
+          <TrackTile
+            uid='test-uid'
+            id={1}
+            index={0}
+            size={TrackTileSize.SMALL}
+            statSize='small'
+            ordered={false}
+            togglePlay={() => {}}
+            isLoading={false}
+            hasLoaded={() => {}}
+            isTrending={false}
+            isFeed={false}
+          />
+        }
+      />
+      <Route path='/test-user/test-track' element={<h1>Mock Track Page</h1>} />
+      <Route path='/test-user' element={<h1>Mock User Page</h1>} />
+    </Routes>
+  )
+}
+
+describe('TrackTile', () => {
+  beforeAll(() => {
+    mswServer.listen()
+  })
+
+  afterEach(() => {
+    mswServer.resetHandlers()
+  })
+
+  afterAll(() => {
+    mswServer.close()
+  })
+
+  it('rendersnon-owner track tile with title and user', async () => {
```

Comment:
nit: grammar clean up here
```suggestion
  it('Renders non-owner track tile with title and user', async () => {
```